satellite/rewards/partners_service.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package rewards
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"path"
+
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+)
+
+var (
+	// Error is the default error class for partners package.
+	Error = errs.Class("partners error class")
```

Comment:
nit: I see odd to read `class` in an error message.

For example `"partners error class: name world already exists"`.

Reply:
I agree.